diff --git a/PintOS/src/utils/Pintos.pm b/PintOS/src/utils/Pintos.pm
index 70df40d..1dab6ee 100644
--- a/PintOS/src/utils/Pintos.pm
+++ b/PintOS/src/utils/Pintos.pm
@@ -359,7 +359,7 @@ sub cyl_sectors {
 # Makes sure that the loader is a reasonable size.
 sub read_loader {
     my ($name) = @_;
-    $name = find_file ("loader.bin") if !defined $name;
+    $name = find_file ("/home/dcipher/code/os/chrono-paper/PintOS/src/threads/build/loader.bin") if !defined $name;
     die "Cannot find loader\n" if !defined $name;
 
     my ($handle);
diff --git a/PintOS/src/utils/pintos b/PintOS/src/utils/pintos
index c0da5a3..2ce3763 100755
--- a/PintOS/src/utils/pintos
+++ b/PintOS/src/utils/pintos
@@ -256,7 +256,7 @@ sub set_disk {
 sub find_disks {
     # Find kernel, if we don't already have one.
     if (!exists $parts{KERNEL}) {
-	my $name = find_file ('kernel.bin');
+	my $name = find_file ('/home/dcipher/code/os/chrono-paper/PintOS/src/threads/build/kernel.bin');
 	die "Cannot find kernel\n" if !defined $name;
 	do_set_part ('KERNEL', 'file', $name);
     }
